src/components/EnergySeriesChart.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useRef } from 'react';
+import Chart from 'chart.js';
+import { EnergySeries } from 'types/api';
+import { getChartLineColors } from 'utils/chartUtils';
+
+interface OwnProps {
+  data: EnergySeries[];
+}
+
+//TODO: memoize chart to prevent from re-rendering
+export const EnergySeriesChart = ({ data }: OwnProps) => {
+  const canvasRef = useRef<HTMLCanvasElement>(null);
+
+  useEffect(() => {
+    const context = canvasRef?.current?.getContext('2d');
+
+    //Doesn't render chart when no series data or canvas.
+    if (!context || data.length < 1) return;
+
+    const labels = Object.entries(data[0]).map(([key]) => key);
+
+    const datasets = data.map((series, index) => {
+      return {
+        data: Object.entries(series).map(([_, value]) => value),
+        borderColor: getChartLineColors(index),
+        pointBackgroundColor: getChartLineColors(index),
+        fill: false,
+        borderWidth: 1.5,
+        pointHitRadius: 4,
+        pointBorderWidth: 0.1,
+        pointRadius: 1.5,
+        pointHoverRadius: 1.5,
+      };
+    });
+
+    const options = {
```

Comment:
Just an initial styling for the chart. Looking something like this right now. 
![image](https://user-images.githubusercontent.com/42070238/108736599-1c383080-7575-11eb-85ab-7e1ad387d1c5.png)

I will pollish the chart more in future pull requests. 


Removed xAxis labels to save vertical space. Y range starts always from 0 and higher value is calculated automatically based on passed values. Hovering graph points displays date and value information on a tooltip. Playing to make the table a little bit wider.

Feedback welcomed.

Reply:
Style looks good. No x axis labels is fine, since eventually I plan to modify the backend in such a way that all energy series have data for the same period of time (e.g., from 6am to 7pm). Y axis labels is absolutely needed though, so please don't remove it.
Hovering action is good, but can be removed if it will improve performance.